src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -52,6 +53,54 @@ const SaleMessage: React.FC<SaleMessageProps> = ({ saleMessage }) => {
   )
 }
 
+interface OfferDisplayProps {
+  originalPrice: string | null | undefined
+  offerPrice: string | null | undefined
+  endAt?: string | null
+  isAvailable?: boolean | null
+}
+
+const OfferDisplay: React.FC<OfferDisplayProps> = ({
```

Comment:
minor: For organization I'd move this component [down with the others below the main export](https://github.com/artsy/force/pull/13660/files#diff-7e67206208c67ae3f411657c84fa58ff49ccc4f44682f608a1ce8d56b46d4720R406).